tools/wptrunner/wptrunner/executors/testharness_webdriver_resume.js

```diff
@@ -1,3 +1,24 @@
 var callback = arguments[arguments.length - 1];
-window.opener.testdriver_callback = callback;
-window.opener.process_next_event();
+
+window.addEventListener("message", function handle(event) {
```

Comment:
I don't like the pattern where we keep adding and removing message listeners; we've had bags with such code before. Ideally we define the message handler once in `testharnessreport.js` and just have this script update the current callback and resume the test.